src/components/game/Phase1.js

```diff
@@ -59,24 +73,43 @@ const Phase1 = ({
                 value={playerInput[index] || ''}
                 onChange={(event) => handleInputChange(index, event, inputRefs)}
                 onKeyDown={(event) => handleBackspaceNavigation(index, event)}
-                className="w-full aspect-square rounded-lg font-bold text-center bg-sp-white text-sp-black p-0 
-                  max-w-20 sm:max-w-24 md:max-w-28 lg:max-w-32 text-2xl md:text-3xl lg:text-4xl xl:text-5xl 2xl:text-6xl"
+                className={
+                  'w-full aspect-square rounded-lg font-bold text-center bg-sp-white text-sp-black p-0 max-w-20 sm:max-w-24 md:max-w-28 lg:max-w-32 text-2xl md:text-3xl lg:text-4xl xl:text-5xl 2xl:text-6xl'
+                }
+                style={
+                  incorrectIndices.includes(index)
+                    ? {
+                        backgroundColor: 'rgb(242 140 140)',
+                        border: '4px solid rgb(227, 17, 48)',
+                      }
+                    : { backgroundColor: 'rgb(255 255 255)' }
+                }
                 autoCapitalize="none"
+                disabled={inputDisabled}
               />
             ))}
           </div>
-          <div className="flex items-end justify-center sm:justify-end  py-2">
-            <button
-              className={`btn-sp-primary w-full sm:w-1/2 ${
-                isReadyButtonDisabled
-                  ? 'bg-sp-gray cursor-not-allowed'
-                  : 'bg-sp-light-green cursor-pointer'
-              }`}
-              onClick={handleSubmit}
-              disabled={isReadyButtonDisabled}
-            >
-              VALMIS
-            </button>
+          <div className="flex items-end justify-center sm:justify-end py-2">
+            {showContinueButton ? (
+              <button
+                className="btn-sp-primary w-full sm:w-1/2 bg-sp-light-green cursor-pointer"
+                onClick={handleContinueOnWrongAnswer}
+              >
+                JATKA PELIÄ
```

Comment:
Vaihda teksti "JATKA" kaikissa phaseissa.

Reply:
Nyt tässä pitäisi olla kaikki ok myös! Korjasin tuon skippauksen logiikan niin, että wordAttempts muuttajaa kasvatetaan yhdellä aina, kun 1-vaihe menee väärin. showSkipButtonille asetetaan arvo true, jos wordAttempts on yhtä suuri tai suurempi kuin 2. Toteutin tämän näin, koska ainoa ehto skipButtonin näyttämiselle on, että pelin toisessa vaiheessa käydään toista kertaa. Tällöin 1-vaiheen on mentävä väärin vähintään 2 kertaa, jotta skippaukselle annetaan mahdollisuus (koska jos 1-vaihe menee oikein yhdenkin kerran siirrytään jo seuraavana sanaan). Aiemmin skippaus ei toiminut esim. tilanteessa, jossa vaiheet 2 ja 3 menevät oikein, mutta vaihe 1 menee kaksi kertaa väärin. Nyt skippauksen pitäisi toimia halutusti riippumatta, siitä kuinka vaiheet 2 ja 3 menevät.